.evergreen/run-tests.sh

```diff
@@ -34,33 +34,44 @@ export DRIVER_TOOLS_CA_PEM="${DRIVERS_TOOLS}/.evergreen/x509gen/ca.pem"
 export DRIVER_TOOLS_CLIENT_KEY_ENCRYPTED_PEM="${DRIVERS_TOOLS}/.evergreen/x509gen/password_protected.pem"
 
 
-source ~/.rvm/scripts/rvm
-
 # Necessary for jruby
 export JAVACMD=/opt/java/jdk8/bin/java
 export PATH=$PATH:/opt/java/jdk8/bin
 
 if [ "$RVM_RUBY" == "ruby-head" ]; then
-  rvm reinstall $RVM_RUBY
-fi
-
-# Don't errexit because this may call scripts which error
-set +o errexit
-rvm use $RVM_RUBY
-set -o errexit
-
-# Ensure we're using the right ruby
-python - <<EOH
+  # 12.04, 14.04 and 16.04 are good
+  wget -O ruby-head.tar.bz2 http://rubies.travis-ci.org/ubuntu/`lsb_release -rs`/x86_64/ruby-head.tar.bz2
+  tar xf ruby-head.tar.bz2
+  export PATH=`pwd`/ruby-head/bin:`pwd`/ruby-head/lib/ruby/gems/2.6.0/bin:$PATH
+  ruby --version
+  ruby --version |grep dev
```

Comment:
Just to make sure I understand, is the purpose of this line just to exit the script if `dev` is not found?

Reply:
That is correct.